### ISSUE TITLE
undefined `__builtin_ia32_pcmpestri128` on ldc 1.32

### ISSUE THREAD

**Issue by skoppe** [tchaloupka/httparsed#4]
ldc 1.30 works, but on 1.32 I get this error:

```
../../../.dub/packages/httparsed-1.2.1/httparsed/source/httparsed.d(973,26): Error: undefined identifier `__builtin_ia32_pcmpestri128`
```

**Comment by tchaloupka** [comment]
Hi, sorry for late reply.
I'm just trying it and it seems to be a fixed problem in [ldc-1.32.1](https://github.com/ldc-developers/ldc/releases/tag/v1.32.1).
At least with `dub test` it fails for me too with `1.32.0` but is ok with `1.32.1`.

See also: https://github.com/ldc-developers/ldc/issues/4356

**Comment by skoppe** [comment]
Ah, good. Thank for looking into it.
